test/jsonInt64.spec.js

```diff
@@ -1,5 +1,7 @@
-var expect = require('chai').expect;
-var Assertion = require('chai').Assertion;
+var chai = require('chai');
+var Assertion = chai.Assertion;
```

Comment:
I am not seeing ``Assertion`` used anywhere in this changeset - is it really needed?

Reply:
I use it to add a custom equality checker for `Int64`